axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/LegacyAxonServerEventStorageEngine.java

```diff
@@ -103,23 +109,12 @@ private static String resolveAggregateType(Set<Tag> tags) {
     @Override
     public CompletableFuture<AppendTransaction> appendEvents(@Nonnull AppendCondition condition,
                                                              @Nonnull List<TaggedEventMessage<?>> events) {
-        if (condition.criteria().tags().size() > 1) {
```

Comment:
I am just now wondering, but removing this validation means any number of models could be updated in a single go. So far so good. But, that means Legacy Axon Server's `AppendEventsTransaction` already allows this. Otherwise I would expect the tests to fail.

Does that mean legacy Axon Server "already" validates any number of aggregate sequences you pass it in a single transaction?

Reply:
Yes, Axon Server already allows for multiple aggregates to be updated in a single transaction.

Reply:
And that's how you keep learning every day.